samples/sampler/stories/qa/Spotlight.js

```diff
@@ -291,6 +295,28 @@ export default {
 	component: 'Spotlight'
 };
 
+const SimpleDiv = () => {
+	action('Render')(true);
+	return <div>Spottable Component</div>;
+};
+
+const SpottableItem = Spottable(Skinnable('div'));
+export const CheckRerender = () => (
+	<div>
+		<p>
+			A spottable component must not be re-rendered when a focus change occurs.
+			So the message of the Actions tab(&apos;Render: true&apos;) should be displayed only once when it is first loaded.
```

Comment:
Let's remove `when it is first loaded` since the original issue occurred when the focus moves in and out.